lib/game_session.rb

```diff
@@ -0,0 +1,79 @@
+# this class will make orchestrate interactions between all classes
+class GameSession
+
+  attr_accessor :player1, :player2, :board, :rules, :display, :translator, :current_player
+
+  def initialize(player1, player2, board, rules, display)
+    @player1 = Player.new("X")
+    @player2 = Player.new("O")
+    @board = Board.new(3, 3)
+    @rules = Rules.new(@board)
+    @display = Display.new(@board)
+    @translator = setup_translator
+    @current_player = @player1
+    # binding.pry
+    self.start_game
+
+  end
+
+  def start_game
+    p "Welcome to TicTacToe! Player 1 is 'X' and Player 2 is 'O'."
```

Comment:
display_board should output a neatly formatted stringified board (including the new lines to separate the rows). Using either print or puts will achieve this. Ill use print to remain consistent with the methods mentioned below.

start_game and provide_move will be using print (instead of p or puts) because the desired output should not include an appended new line after the string messages.